examples/client/common.c

```diff
@@ -743,23 +754,243 @@ int ClientUseCert(const char* certName, void* heap)
     return ret;
 }
 
+#ifdef WOLFSSH_TPM
+
+static int readKeyBlob(const char* filename, WOLFTPM2_KEYBLOB* key)
+{
+    int rc = 0;
+#if !defined(NO_FILESYSTEM) && !defined(NO_WRITE_TEMP_FILES)
+    XFILE  fp = NULL;
```

Comment:
Please switch to using the wolfSSH file porting macros `WFILE` and `WFOPEN`. This is using the wolfSSL ones.